common/changes/shift-cli/cli-spinner_2019-08-13-11-37.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
```

Comment:
Note that an empty changelog will not get the changes to the package published until the next non-empty changelog.

Reply:
ack